x-pack/elastic-agent/pkg/agent/cmd/run.go

```diff
@@ -135,21 +136,27 @@ func run(streams *cli.IOStreams, override cfgOverrider) error {
 
 	statusCtrl := status.NewController(logger)
 
+	// TODO: How do we get the version?
+	version := release.Version()
+	tracer, err := apm.NewTracer(agentName, version)
```

Comment:
We should call `apm.DefaultTracer.Close()` somewhere if we're not using it.

Reply:
do you have a recommendation between using the default tracer, or a new one?

Reply:
Creating a new one is appropriate since you're configuring it via code. The default one is appropriate when configuring externally with environment variables.